server/lib/nl/nl_data_spec_next.py

```diff
@@ -209,8 +209,12 @@ def populate_simple(uttr: Utterance) -> bool:
 def _populate_simple_cb(state: PopulateState, chart_vars: ChartVars,
                         places: List[Place],
                         chart_origin: ChartOriginType) -> bool:
-  return _add_chart_to_utterance(ChartType.TIMELINE_CHART, state, chart_vars,
-                                 places, chart_origin)
+  if len(chart_vars.svs) <= _MAX_VARS_PER_CHART:
```

Comment:
nit: use if condition to set the chart_type? That makes it clear the only difference is chart type

Reply:
Good idea.